pyworkflow/gui/form.py

```diff
@@ -1647,13 +1648,22 @@ def _addButton(text, icon, command, col):
                            compound=tk.LEFT, cursor='hand2', name=text.lower())
             btn.bind('<Button-1>', command)
             btn.grid(row=0, column=col, padx=5, sticky='e')
-        
+
         _addButton(pwutils.Message.LABEL_CITE,
                    pwutils.Icon.ACTION_REFERENCES,
                    self._showReferences, 2)
         _addButton(pwutils.Message.LABEL_HELP,
                    pwutils.Icon.ACTION_HELP, self._showHelp, 3)
-        
+
+        # Add protocol url
+        url = plugin.getClassPlugin().getUrl(prot)
+        # If not empty...
+        if url:
```

Comment:
Why to add another button? Can we put the "URL" of the protocol inside the already existing "Help" content?

Reply:
I've moved it inside Help info page as suggested.

Reply:
Thanks @pconesa for taking my suggestion into account.